docs/tutorials/hpc-slurm-qwiklabs.yaml

```diff
@@ -31,55 +31,53 @@ deployment_groups:
   # Source is an embedded module, denoted by "modules/*" without ./, ../, /
   # as a prefix. To refer to a local module, prefix with ./, ../ or /
   # Example - ./modules/network/vpc
-  - id: network1
+  - id: network
     source: modules/network/vpc
 
-  - id: debug_node_group
-    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+  - id: debug_nodeset
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network]
     settings:
       node_count_dynamic_max: 4
-      machine_type: n2d-standard-2
+      machine_type: n2-standard-2
```

Comment:
Is there a reason you are changing from n2d to n2. I believe n2d was selected to align with lab quotas.

Reply:
Updated.